OCR/README.md

```diff
@@ -7,77 +7,95 @@ pipx install poetry
 ```
 
 Activate the virtual environment and install dependencies, all subsequent commands assume you are in the virtual env
+
 ```shell
 poetry shell
 poetry install
 ```
 
 Run unit tests
+
 ```shell
 poetry run pytest
 ```
 
 Run benchmark tests
+
 ```shell
 cd tests
 poetry run pytest benchmark_test.py -v
 ```
 
-poetry run pytest bench_test.py -v  
+poetry run pytest bench_test.py -v
 
 Run main, hoping to convert this to a cli at some point
+
 ```shell
 poetry run main
 ```
 
 To build the OCR service into an executable artifact
+
 ```shell
 poetry run build
 ```
 
 Adding new dependencies
+
 ```shell
 poetry add package-name
 ```
 
 To manually update the poetry lock file
+
 ```shell
 poetry lock
 ```
 
 To view installed packages in the virtual env
+
 ```shell
 poetry show
 ```
 
 To lint your files using ruff
+
 ```shell
 ruff check --fix
 ```
 
-To format your files using ruff 
+To format your files using ruff
+
 ```shell
 ruff format
 ```
 
 To run the API in dev mode with reload
+
 ```shell
 fastapi dev ocr/api.py
 ```
 
 To run the API in prod mode
+
 ```shell
 poetry run api
 ```
 
+### Test Data Sets
+
+Here is the standarized form testing dataset
+https://drive.google.com/drive/folders/1WS2FYn0BTxWv0juh7lblzdMaFlI7zbDd
```

Comment:
Nit: This is a public repo and that google drive shouldn't be public. I'd personally just remove this entirely and let folks run the script to download them.

Reply:
I removed it. The script should do the data pull.